docs/Bed_Mesh.md

```diff
@@ -410,17 +415,34 @@ adaptive_margin: 5
 
   ![adaptive_bedmesh_margin](img/adaptive_bed_mesh_margin.svg)
 
-By nature, adaptive bed meshes use the objects defined by the Gcode file being printed.
-Therefore, it is expected that each Gcode file will generate a mesh that probes a different
-area of the print bed. Therefore, adapted bed meshes should not be re-used. The expectation
+#### Caveats:
+
+By nature, adaptive bed meshes use the objects defined by the [exclude_object](Exclude_Object.md) module
+in the Gcode file being printed, and it is expected that each Gcode file will generate a mesh that probes a different
+area of the print bed. Therefore, adapted bed meshes should **not** be re-used. The expectation
 is that a new mesh will be generated for each print if adaptive meshing is used.
 
-It is also important to consider that adaptive bed meshing is best used on machines that can
+It is important to consider that adaptive bed meshing is best used on machines that can
 normally probe the entire bed and achieve a maximum variance less than or equal to 1 layer
 height. Machines with mechanical issues that a full bed mesh normally compensates for may
-have undesirable results when attempting print moves **outside** of the probed area. If a
-full bed mesh has a variance greater than 1 layer height, caution must be taken when using
-adaptive bed meshes and attempting print moves outside of the meshed area.
+have undesirable results when attempting print moves **outside** of the probed area. If you
```

Comment:
I generally prefer documentation be written in a "neutral" way, without using pronouns. Not sure how to explain it so I'll give an example:

Instead of "If you normally have ...", say "If a printer normally has ...".